### DISCUSSION TITLE
Does anyone know how to translate from "csgo::pod::ConVar *" to "ConVar *"

### DISCUSSION THREAD

**Discussion by PEXCH-SENTXMENT** [danielkrupinski/Osiris#3906] [Q&A]
Need to get the values of "sv_accelerate" and "sv_maxspeed"

Severity Code Description Project File String Suppression status
Error (active) E0513 value of type "csgo::pod::ConVar *" cannot be assigned to an entity of type "ConVar *" cmake_pch.cxx.obj (CMakeFiles\OsirisOBJ.dir\cmake_pch.cxx.obj) - x64-Debug C:\Users\peeea\source\repos\Osiris2\Source\Hacks\Aimbot.cpp 310

**Comment by Xsintashi** [comment]
give snippet of the code. I have no idea what's going on

**Reply by PEXCH-SENTXMENT** [reply]
I solved the problem
